README.rst

```diff
@@ -50,21 +51,25 @@ Live tests (triggered manually)                        |workflow livetests badge
 .. |fair-software badge| image:: https://img.shields.io/badge/fair--software.eu-%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F-green
    :target: https://fair-software.eu
    
-.. |ReadTheDocs Badge| image:: https://readthedocs.org/projects/howfairis/badge/?version=latest
+.. |readthedocs badge| image:: https://readthedocs.org/projects/howfairis/badge/?version=latest
    :target: https://howfairis.readthedocs.io/en/latest/?badge=latest
    :alt: Documentation Status
    
 .. |python versions badge| image:: https://img.shields.io/pypi/pyversions/howfairis.svg
    :target: https://pypi.python.org/pypi/howfairis   
 
-.. |Sonarcloud quality badge| image:: https://sonarcloud.io/api/project_badges/measure?project=fair-software_howfairis&metric=alert_status
+.. |sonarcloud quality badge| image:: https://sonarcloud.io/api/project_badges/measure?project=fair-software_howfairis&metric=alert_status
    :target: https://sonarcloud.io/dashboard?id=fair-software_howfairis
    :alt: Quality Gate Status
 
-.. |Sonarcloud coverage badge| image:: https://sonarcloud.io/api/project_badges/measure?project=fair-software_howfairis&metric=coverage
+.. |sonarcloud coverage badge| image:: https://sonarcloud.io/api/project_badges/measure?project=fair-software_howfairis&metric=coverage
    :target: https://sonarcloud.io/dashboard?id=fair-software_howfairis
    :alt: Coverage
 
+.. |dockerhub badge| image:: https://img.shields.io/docker/build/fairsoftware/howfairis
+   :target: https://hub.docker.com/repository/docker/fairsoftware/howfairis
```

Comment:
Used url is not public.
```suggestion
   :target: https://hub.docker.com/r/fairsoftware/howfairis
```

Reply:
didnt even know that, thanks

Reply:
Hmm, badge still says `repo not found`, I hoped a URL correction would help. Needs some more attention.